release-notes/7.0/preview/api-diff/preview4/Microsoft.NETCore.App/7.0-preview4_System.Runtime.InteropServices.md

```diff
@@ -0,0 +1,248 @@
++        static NFloat System.Numerics.IBitwiseOperators<System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat>.op_BitwiseAnd(NFloat left, NFloat right);
++        static NFloat System.Numerics.IBitwiseOperators<System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat>.op_BitwiseOr(NFloat left, NFloat right);
++        static NFloat System.Numerics.IBitwiseOperators<System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat>.op_ExclusiveOr(NFloat left, NFloat right);
++        static NFloat System.Numerics.IBitwiseOperators<System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat>.op_OnesComplement(NFloat value);
++        static NFloat System.Numerics.IDecrementOperators<System.Runtime.InteropServices.NFloat>.op_CheckedDecrement(NFloat value);
++        static NFloat System.Numerics.IDivisionOperators<System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat>.op_CheckedDivision(NFloat left, NFloat right);
++        int System.Numerics.IFloatingPoint<System.Runtime.InteropServices.NFloat>.GetExponentByteCount();
++        long System.Numerics.IFloatingPoint<System.Runtime.InteropServices.NFloat>.GetExponentShortestBitLength();
++        long System.Numerics.IFloatingPoint<System.Runtime.InteropServices.NFloat>.GetSignificandBitLength();
++        int System.Numerics.IFloatingPoint<System.Runtime.InteropServices.NFloat>.GetSignificandByteCount();
++        bool System.Numerics.IFloatingPoint<System.Runtime.InteropServices.NFloat>.TryWriteExponentLittleEndian(Span<byte> destination, out int bytesWritten);
++        bool System.Numerics.IFloatingPoint<System.Runtime.InteropServices.NFloat>.TryWriteSignificandLittleEndian(Span<byte> destination, out int bytesWritten);
++        static NFloat System.Numerics.IIncrementOperators<System.Runtime.InteropServices.NFloat>.op_CheckedIncrement(NFloat value);
++        static NFloat System.Numerics.IMultiplyOperators<System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat>.op_CheckedMultiply(NFloat left, NFloat right);
++        static NFloat System.Numerics.ISubtractionOperators<System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat>.op_CheckedSubtraction(NFloat left, NFloat right);
++        static NFloat System.Numerics.IUnaryNegationOperators<System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat>.op_CheckedUnaryNegation(NFloat value);
++        static NFloat Tan(NFloat x);
++        static NFloat Tanh(NFloat x);
++        static NFloat Truncate(NFloat x);
++        static bool TryCreate<TOther>(TOther value, out NFloat result);
```

Comment:
Notable differences from other diffs include that explicitly implemented operators aren't correctly formatted
```suggestion
+        public static NFloat Pow(NFloat x, NFloat y);
+        public static NFloat ReciprocalEstimate(NFloat x);
+        public static NFloat ReciprocalSqrtEstimate(NFloat x);
+        public static NFloat Round(NFloat x);
+        public static NFloat Round(NFloat x, int digits);
+        public static NFloat Round(NFloat x, int digits, MidpointRounding mode);
+        public static NFloat Round(NFloat x, MidpointRounding mode);
+        public static NFloat ScaleB(NFloat x, int n);
+        public static int Sign(NFloat value);
+        public static NFloat Sin(NFloat x);
+        public static (NFloat Sin, NFloat Cos) SinCos(NFloat x);
+        public static NFloat Sinh(NFloat x);
+        public static NFloat Sqrt(NFloat x);
+        static NFloat System.Numerics.IAdditionOperators<System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat>.operator checked +(NFloat left, NFloat right);
+        static NFloat System.Numerics.IBitwiseOperators<System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat>.operator &(NFloat left, NFloat right);
+        static NFloat System.Numerics.IBitwiseOperators<System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat>.operator |(NFloat left, NFloat right);
+        static NFloat System.Numerics.IBitwiseOperators<System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat>.operator ^(NFloat left, NFloat right);
+        static NFloat System.Numerics.IBitwiseOperators<System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat>.operator ~(NFloat value);
+        static NFloat System.Numerics.IDecrementOperators<System.Runtime.InteropServices.NFloat>.op_CheckedDecrement(NFloat value);
+        static NFloat System.Numerics.IDivisionOperators<System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat>.operator checked /(NFloat left, NFloat right);
+        int System.Numerics.IFloatingPoint<System.Runtime.InteropServices.NFloat>.GetExponentByteCount();
+        long System.Numerics.IFloatingPoint<System.Runtime.InteropServices.NFloat>.GetExponentShortestBitLength();
+        long System.Numerics.IFloatingPoint<System.Runtime.InteropServices.NFloat>.GetSignificandBitLength();
+        int System.Numerics.IFloatingPoint<System.Runtime.InteropServices.NFloat>.GetSignificandByteCount();
+        bool System.Numerics.IFloatingPoint<System.Runtime.InteropServices.NFloat>.TryWriteExponentLittleEndian(Span<byte> destination, out int bytesWritten);
+        bool System.Numerics.IFloatingPoint<System.Runtime.InteropServices.NFloat>.TryWriteSignificandLittleEndian(Span<byte> destination, out int bytesWritten);
+        static NFloat System.Numerics.IIncrementOperators<System.Runtime.InteropServices.NFloat>.operator checked ++(NFloat value);
+        static NFloat System.Numerics.IMultiplyOperators<System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat>.operator checked *(NFloat left, NFloat right);
+        static NFloat System.Numerics.ISubtractionOperators<System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat>.operator checked -(NFloat left, NFloat right);
+        static NFloat System.Numerics.IUnaryNegationOperators<System.Runtime.InteropServices.NFloat,System.Runtime.InteropServices.NFloat>.operator checked -(NFloat value);
+        public static NFloat Tan(NFloat x);
+        public static NFloat Tanh(NFloat x);
+        public static NFloat Truncate(NFloat x);
+        public static bool TryCreate<TOther>(TOther value, out NFloat result) where TOther : INumber<TOther>;
```